ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs

```diff
@@ -68,9 +70,23 @@ public class ECMA2YamlConverter
             }
 
             Console.WriteLine("Building loaded files...");
+            Console.WriteLine($"ECMA2YamlRepoConfig:{JsonConvert.SerializeObject(config)}");
             store.UWPMode = config?.UWP ?? false;
             store.Build();
 
+            if (store.UWPMode)
```

Comment:
we don't need to check for UWPMode here, we didn't check in previous version

Reply:
removed